pkg/restapi/operation/operations.go

```diff
@@ -13,6 +13,8 @@ import (
 	"io/ioutil"
 	"net/http"
 
+	"github.com/google/uuid"
+
```

Comment:
All the external library imports go together - remove this extra line break on line 17